web/src/pages/guidance/indevelopment.page.tsx

```diff
@@ -60,6 +60,8 @@ const tableBodyRender = (documents: Document[]) => (
 );
 
 export default getGuidanceListPage({
+	metaDescription:
+		"View a complete list of all our guidance, NICE advice and quality standards currently in development",
```

Comment:
Some difference in language used versus line 58 "View" versus "See" ?

Reply:
Both should probably just say "A complete list of..."

Reply:
Ahhhh yes that was deliberate - see my comment on the Jira card with Jane's comment